pyface/action/group.py

```diff
@@ -13,14 +13,14 @@
 from functools import partial
 
 
-from traits.api import Any, Bool, HasTraits, Instance, List, Property
+from traits.api import Any, Bool, HasTraits, List, Property
 from traits.api import Str
 from traits.trait_base import user_name_for
 
 
 from pyface.action.action import Action
 from pyface.action.action_item import ActionItem
-from pyface.action.action_manager_item import ActionManagerItem
+from pyface.action.action_manager_item import ActionManagerItem  # noqa: F401
```

Comment:
Why are you keeping this here? Did you find code which imports `ActionManagerItem` from this module?

Reply:
I kept this around because `group._items` should be `List(ActionManagerItem)` but at the moment, its just defined as `List`. I'll remove it from this PR - and test in a separate PR what happens if i correct the trait definition.